actionpack/lib/action_controller/caching.rb

```diff
@@ -9,17 +9,16 @@ module ActionController
   # of calculations, renderings, and database calls around for subsequent
   # requests.
   #
-  # You can read more about each approach by clicking the modules below.
-  #
   # Note: To turn off all caching provided by Action Controller, set
+  #
   #     config.action_controller.perform_caching = false
   #
   # ## Caching stores
   #
-  # All the caching stores from ActiveSupport::Cache are available to be used as
+  # All the caching stores from ActiveSupport::Cache::Store are available to be used as
   # backends for Action Controller caching.
   #
-  # Configuration examples (FileStore is the default):
+  # Configuration examples (defaults to ActiveSupport::Cache::MemoryStore when not set):
```

Comment:
Incorrect, because https://github.com/rails/rails/blob/133ed38df02f93ebdaf9821219c4124db76beeb9/railties/lib/rails/application/configuration.rb#L57

Reply:
I see ... I was looking at a combination of https://github.com/rails/rails/pull/52434 and https://github.com/rails/rails/blob/v8.0.1/activesupport/lib/active_support/cache.rb#L62-L63 ... but do see now that if no setting at all in env file, file_store is chosen. Removed this portion as well.